routers/api/v1/convert/convert.go

```diff
@@ -10,11 +10,32 @@ import (
 	"github.com/Unknwon/com"
 
 	api "github.com/gogits/go-gogs-client"
+	"github.com/gogits/git-module"
 
 	"github.com/gogits/gogs/models"
 	"github.com/gogits/gogs/modules/setting"
 )
 
+// ToApiBranch converts user to its API format.
```

Comment:
This part of code should put after at least later then `ToApiRepository` as `branch` belongs to a `repository`.